cmd/notary/integration_test.go

```diff
@@ -1025,6 +1066,235 @@ func TestClientKeyImportExportRootOnly(t *testing.T) {
 		t, tempDir, server.URL, "gun", target, tempFile.Name())
 }
 
+// Tests importing and exporting keys for all different roles and GUNs
+func TestClientKeyImportExportAllRoles(t *testing.T) {
```

Comment:
The loop sounds like a great idea, we'll have conditionals for each role type for the right subdirs but I think it'll still be better.

And yeah, also a good idea to remove the key -- I don't think the test is reading the previous import, but it'd be good to make it explicit.